### PR TITLE
test: Don't upload built rpms to log sink

### PR DESCRIPTION
**Pull request by dperpeet**
These take up a significant amount of disk space for little gain.

### CODE REVIEW

File: bots/image-prepare [hunk 1]
```diff
@@ -68,10 +68,7 @@ def main():
         return 2
 
     # Default to putting build output in bots directory
-    if "TEST_ATTACHMENTS" in os.environ:
-        results = os.path.join(os.environ["TEST_ATTACHMENTS"], "build-results")
-    else:
-        results = os.path.join(BOTS, "build-results")
+    results = os.path.join(BOTS, "build-results")
```
Comment (stefwalter) [inline bots/image-prepare]: I would suggest something like this: ```os.path.join(os.environ.get("TEST_DATA", BASE), "tmp", "build-results")```

Comment (dperpeet) [inline bots/image-prepare]: changed - I don't have a strong opinion either way

File: bots/image-prepare [hunk 2]
```diff
@@ -67,11 +67,8 @@ def main():
         sys.stderr.write("image-prepare: use of --install-only with --build-only is incompatible\n")
         return 2
 
-    # Default to putting build output in bots directory
-    if "TEST_ATTACHMENTS" in os.environ:
-        results = os.path.join(os.environ["TEST_ATTACHMENTS"], "build-results")
-    else:
-        results = os.path.join(BOTS, "build-results")
+    # Default to putting build output in the TEST_DATA directory
+    results = os.path.join(os.environ.get("TEST_DATA", BASE), "tmp", "build-results")
```
Comment (stefwalter) [inline bots/image-prepare]: Cool. How did you test this?

Comment (dperpeet) [inline bots/image-prepare]: with image-prepare
```
TZ=EST TEST_OS=fedora-25 ./bots/image-prepare -v
```

the rpms are put into the cockpit-data dir:

```
[admin@localhost cockpit-data]$ find . -name "*.rpm" -type f
./tmp/build-results/cockpit-141.x-1.wip.fc25.src.rpm
```

**Pull request merged**